brooklinevoiceapp/mycity/utils/brookline_arcgis_api_utils.py

```diff
@@ -111,58 +116,55 @@ def get_nearest_feature_json(address: str,
         INSR_PARAM: "102100",
         OUT_FIELDS_PARAM: "*",
         OUTSR_PARAM: "102100",
-        GEOMETRY_PARAM: json.dumps(_geocode_address(address))
+        GEOMETRY_PARAM: json.dumps(home_address)
```

Comment:
This needs to be changed to:
`GEOMETRY_PARAM: json.dumps([home_address['x'], home_address['y']])`

the reason being is coordinates being passed aren't in the format they need to be. This fixes that issue.

Reply:
This should fix it according to my testing.